src/os/structs/nodeutils.js

```diff
@@ -0,0 +1,21 @@
+goog.provide('os.structs.nodeutils');
```

Comment:
We dropped the "*utils" convention a while back. Utilities for arrays are on `os.array`, object are on `os.object`, etc. I think you can do `os.structs` or `os.structs.treenode` and that would fit better in that convention.

Reply:
Fixed